lib/custom_a11y_rules.js

```diff
@@ -116,7 +116,8 @@ var customRules = {
                     'fail': 'all <nav> elements MUST have aria-label or aria-labelledby attribute'
                 }
             },
-            evaluate: function(node, options) {
+            evaluate: function(node, option) {
```

Comment:
Did this need to be changed?  It looks like all the other checks still use "options".

Reply:
woops. good catch! That was accidental.